roles/lib_openshift_api/build/src/base.py

```diff
@@ -1,7 +1,14 @@
-#!/usr/bin/env python
+# pylint: skip-file
 '''
   OpenShiftCLI class that wraps the oc commands in a subprocess
+
+
+#################################
+#  THIS MODULE IS AUTOGENARATED #
+#################################
```

Comment:
I want the autogenerated message to be more "in your face."

Consider doing something like this:
https://github.com/openshift/openshift-tools/blob/master/docker/oso-ops-base/centos7/build.sh#L2-L9